doc/src/cfgrammar.md

```diff
@@ -2,4 +2,4 @@
 
 `cfgrammar` reads in grammar files, processes them, and provides a convenient
 API for operating with them. It may be of interest to those manipulating
-grammars directly.
+grammars directly, or who to wish custom types of parsers.
```

Comment:
`or wish to use custom types of parsers`?

Reply:
Oops!